src/audio/base_fw.c

```diff
@@ -461,7 +466,9 @@ static int basefw_get_large_config(struct comp_dev *dev,
 	/* TODO: add more support */
 	case IPC4_DSP_RESOURCE_STATE:
 	case IPC4_NOTIFICATION_MASK:
+		break;
 	case IPC4_MODULES_INFO_GET:
```

Comment:
The commit message can use some update.
"Added support for ipc4 query no 9." 
->
"Add support to handle IPC4_MODULES_INFO_GET message."

Reply:
Fixed